### PR TITLE
add control for CRISPR (or other gene perturbation types)

### PR DESCRIPTION
**Pull request by allaway**
Adding a gRNA control option for CRISPR experiments to more explicitly describe control samples.

### CODE REVIEW

**Review by nfosi-service — APPROVED** [review]

File: NF.jsonld [hunk 1]
```diff
@@ -18919,6 +18944,23 @@
             ],
             "sms:validationRules": []
         },
+        {
+            "@id": "bts:\"\"",
```
Comment (anngvu) [inline NF.jsonld]: Need to check this. Where is this random new key coming from?

File: NF.csv [hunk 2]
```diff
@@ -724,7 +725,7 @@ compoundName,"Common name for a compound, e.g. “Selumetinib” (https://pubche
...
 compoundDoseUnit,A unit associated with the value(s) in compoundDose.,,,FALSE,Sage Bionetworks,sageCommunity,,,,compoundDoseUnit,Property,Experiment,annotationProperty,experimentProperty,,,
 InChIKey,"A unique hash key for a compound, e.g. for “Selumetinib”, the key is CYOHGALHFOKKQC-UHFFFAOYSA-N (https://pubchem.ncbi.nlm.nih.gov/compound/10127622#section=InChI-Key). This is a more reliable identifier than the compound name and should be used if available.",,,FALSE,Sage Bionetworks,sageCommunity,,,,InChIKey,Property,Experiment,annotationProperty,experimentProperty,,,
-genePerturbationType,Specific way in which a single gene was perturbed in a sample,"overexpression, knockdown, knockout, Not Applicable",,FALSE,Sage Bionetworks,cancer,,,,genePerturbationType,Property,Experiment,annotationProperty,,Gene_Perturbation,,
+genePerturbationType,Specific way in which a single gene was perturbed in a sample,"overexpression, knockdown, knockout, control, Not Applicable",,FALSE,Sage Bionetworks,cancer,,,,genePerturbationType,Property,Experiment,annotationProperty,,Gene_Perturbation,,
```
Comment (anngvu) [inline NF.csv]: Use "non-targeting control"?

Comment (anngvu) [reply NF.csv]: Made change that I suggested in https://github.com/nf-osi/nf-metadata-dictionary/pull/182/commits/aab5ca9c70b184a103add19b643514e54edf27e1...

File: NF.jsonld (hunk 1 above)
Comment (anngvu) [reply NF.jsonld]: Figured out that if there is a space before the "" in valid values, jsonld will contain a key with id `"bts:\"\""`. Resolved with linting in https://github.com/nf-osi/nf-metadata-dictionary/pull/182/commits/05e5deffa3ba894c87c242ce16fbdafefb5fed58

**Comment by anngvu** [comment]
@allaway I made some changes. You can confirm that this is better (?) and merge.

**Review by anngvu — APPROVED** [review]

File: NF.jsonld (hunk 1 above)
Comment (allaway) [reply NF.jsonld]: Interesting, thanks for catching that!

File: NF.csv (hunk 2 above)
Comment (allaway) [reply NF.csv]: Yup...that is what I should be using. Thanks!

**Comment by allaway** [comment]
## NADIA Curation Report

**Synapse Project:** [syn74318745](https://www.synapse.org/#!Synapse:syn74318745)

### Annotations Applied
| Field | Value |
|-------|-------|
| studyName | Early detection of malignant and pre-malignant peripheral nerve sheath tumors |
| diseaseFocus | Neurofibromatosis type 1 |
| manifestation | Malignant Peripheral Nerve Sheath Tumor, Plexiform Neurofibroma |
| dataType | geneExpression |
| assay | rnaSeq |
| species | Homo sapiens |
| tumorType | Malignant Peripheral Nerve Sheath Tumor |
| alternateDataRepository | insdc.sra:PRJNA1136567, insdc.sra:PRJNA1136568 |

### Curation Notes
- RNA-seq for MPNST early detection biomarkers study
- 2 FASTQ files in Raw Data folder
- No PMID found; studyLeads = Not Available

**Status:** pendingReview ✓

**Pull request merged**